src/Jira/Api.php

```diff
@@ -72,6 +72,9 @@ public function __construct(
         AuthenticationInterface $authentication,
         ClientInterface $client = null
     ) {
+        //Regular expression to remove trailing slash
+        $endpoint = preg_replace('{/$}', '', $endpoint);
```

Comment:
Actually just remove it from this PR, because there is another PR doing the same thing: #25 .